Documentation/gettingstarted/istio.rst

```diff
@@ -19,8 +19,8 @@ environment running on your machine.
 
    If running on minikube, you may need to up the memory and CPUs
    available to the minikube VM from the defaults and/or the
-   instructions provided here for the other GSGs. 6GB and 3 CPUs
-   should be enough for Istio (``--memory=6144 --cpus=3``).
+   instructions provided here for the other GSGs. 6GB and 4 CPUs
```

Comment:
Why do we need 4 CPUs now vs. 3?

Reply:
It was needed when I ran the upstream yaml that loaded both v1 and v2 at the same time. We are not using the upstream yaml any more, will check again on 3 cpus (and 5GB memory) to see if tighter resources still work.

Reply:
With 3 CPUs, the step where ratings-v1 and reviews-v2 are added fails to schedule both of them, with this warning in the `kubectl pod describe`:

`  Warning  FailedScheduling  3m (x2 over 3m)  default-scheduler  0/1 nodes are available: 1 Insufficient cpu.`

I'll try again with 4 CPUs and 5GB of memory.

Reply:
4 CPUs works with 5 GB, updated accordingly.